src/Viewer/renderers/mathInput.css

```diff
@@ -0,0 +1,9 @@
+.mathInputWrapper { /* Style the EditableMathField Wrapper */
+    cursor: pointer;
+    margin: 0px 4px 4px 4px;
+    display: inline;
+}
+
+td.matrixCell span div.mathInputWrapper {
```

Comment:
@jaltekruse is the `span` part of this line necessary? Could you explain why `td.matrixCell div.mathInputWrapper {...` wouldn't be sufficient?

Reply:
I was on a branch that made actually testing this a bit hard when I was responding. So I was just looking at the hierarchy of components in production and playing around injecting css with the devtools and then trying to adjust for the changes in the PR when I sent the suggested code, but I hadn't actually tested everything together. Pretty much I was giving approximate code, so I good with whatever works. I thought it was necessary to specify all html elements between the highest thing being targeted and what you actually wanted to modify, is there no longer a span between the two?

Reply:
You don't need to specify all html elements in between if you separate by spaces.  If you include a '>' between the two elements, then it means [direct child](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator), so you can't skip anything.

Reply:
Got it-- I was just curious since I've never used descendant combinators until now! Thanks again for looking into it @jaltekruse!